pkg/cmd/tracee.go

```diff
@@ -13,12 +13,13 @@ import (
 	"github.com/aquasecurity/tracee/pkg/logger"
 	"github.com/aquasecurity/tracee/pkg/server/grpc"
 	"github.com/aquasecurity/tracee/pkg/server/http"
+	"github.com/aquasecurity/tracee/pkg/streams"
 	"github.com/aquasecurity/tracee/pkg/utils"
 )
 
 type Runner struct {
 	TraceeConfig config.Config
-	Printer      printer.EventPrinter
+	Printer      *printer.Broadcast
```

Comment:
Can you please explain why this change is required?

Reply:
Using interface here doesn't communicate well the real intent of Broadcasting. printer.Broadcast has a []EventPrinter and a []config.PrinterConfig.

Reply:
Another reason: retaining the interface would require expanding it with `Kinds()` and `Ignore()` methods, which every implementation would need to define.